net/netfilter/nft_set_pipapo.c

```diff
@@ -852,10 +852,12 @@ static int pipapo_insert(struct nft_pipapo_field *f, const uint8_t *k,
 {
 	int rule = f->rules++, group, ret;
 
-	ret = pipapo_resize(f, f->rules - 1, f->rules);
+	ret = pipapo_resize(f, f->rules, f->rules + 1);
```

Comment:
This is not changed in https://github.com/ctrliq/kernel-src-tree/blob/centos_kernel-4.18.0-511.el8/net/netfilter/nft_set_pipapo.c#L854